.github/ISSUE_TEMPLATE/feature-improvement-request.md

```diff
@@ -1,10 +1,9 @@
 ---
 name: Feature/Improvement request
 about: Suggest an enhancement
-title: ''
+title: ""
```

Comment:
Single quote -> double quote doesn't do much

Reply:
looks nicer tho.

Reply:
eeh i undid the changes by Prettier because the guy who made the whole project didnt like it